MicroAOD/python/flashggPreselectedDiPhotons_cfi.py

```diff
@@ -1,47 +1,82 @@
-                                        ( leadingPhoton.isEB &&
-                                        (leadingPhoton.r9>0.85 || 
-                                        (leadingPhoton.sigmaIetaIeta < 0.015 && 
-                                         leadingPhoton.pfChgIsoWrtChosenVtx02 < 6.0 && 
-                                         leadingPhoton.trkSumPtHollowConeDR03 < 6.0 )))  ||
-                                        ( leadingPhoton.isEE &&
-                                        (leadingPhoton.r9>0.9 || 
-                                        (leadingPhoton.sigmaIetaIeta < 0.035 && 
-                                         leadingPhoton.pfChgIsoWrtChosenVtx02 < 6.0 && 
-                                         leadingPhoton.trkSumPtHollowConeDR03 < 6.0 ))) 
-                                     )
-                                     &&
-                                    (
-                                        ( subLeadingPhoton.isEB &&
-                                        (subLeadingPhoton.r9>0.85 || 
-                                        (subLeadingPhoton.sigmaIetaIeta < 0.015 && 
-                                         subLeadingPhoton.pfChgIsoWrtChosenVtx02 < 6.0 && 
-                                         subLeadingPhoton.trkSumPtHollowConeDR03 < 6.0 )))  ||
-                                        ( subLeadingPhoton.isEE &&
-                                        (subLeadingPhoton.r9>0.9 || 
-                                        (subLeadingPhoton.sigmaIetaIeta < 0.035 && 
-                                         subLeadingPhoton.pfChgIsoWrtChosenVtx02 < 6.0 && 
-                                         subLeadingPhoton.trkSumPtHollowConeDR03 < 6.0 ))) 
-                                     )
-                                        &&
-                                     (leadingPhoton.pt > 14 && leadingPhoton.hadTowOverEm()<0.15 && 
-                                     (leadingPhoton.r9()>0.8 || leadingPhoton.chargedHadronIso()<20 
-                                      || leadingPhoton.chargedHadronIso()<0.3*leadingPhoton.pt())) &&
-                                     (subLeadingPhoton.pt > 14 && subLeadingPhoton.hadTowOverEm()<0.15 &&
-                                     (subLeadingPhoton.r9()>0.8 || subLeadingPhoton.chargedHadronIso()<20 
-                                      || subLeadingPhoton.chargedHadronIso()<0.3*subLeadingPhoton.pt()))               
+phoEffArea=cms.PSet( var=cms.string("abs(superCluster.eta)"), bins=cms.vdouble(0.,0.9,1.5,2,2.2,3), vals=cms.vdouble(0.21,0.2,0.14,0.22,0.31) )
+neuEffArea=cms.PSet( var=cms.string("abs(superCluster.eta)"), bins=cms.vdouble(0.,0.9,1.5,2,2.2,3), vals=cms.vdouble(0.04,0.059,0.05,0.05,0.15) )
+
+PreselectionVariables = cms.vstring(
+    "egChargedHadronIso", 
+    "egPhotonIso", 
+    "egNeutralHadronIso",
+    "hadTowOverEm",
+    "(?r9>0.8||egChargedHadronIso<20||egChargedHadronIso/pt<0.3?full5x5_sigmaIetaIeta:sigmaIetaIeta)",
```

Comment:
Here, I would use directly full5x5_sigmaIetaIeta. The conditional expression was there in the example because of the Phys14 MINIAOD bug.